src/Uno.UWP/Helpers/DrawableHelper.Android.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if __ANDROID__
+using Android.Graphics.Drawables;
+using Android.Support.V4.Content;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.UI;
+
+namespace Uno.Helpers
+{
+	public static class DrawableHelper
+	{
+		private static Dictionary<string, int> _drawablesLookup;
+		private static Type _drawables;		
+		
+		public static Type Drawables
+		{
+			get => _drawables;
+			set
+			{				
+				_drawables = value;
+				InitializeDrawablesLookup();
+			}
+		}
+
+		/// <summary>
+		/// Returns the Id of the bundled image.
+		/// </summary>
+		/// <param name="imageName">Name of the image</param>
+		/// <returns>Resource's id</returns>
+		public static int? FindResourceId(string imageName)
+		{
+			var key = System.IO.Path.GetFileNameWithoutExtension(imageName);
+			if (_drawablesLookup == null)
+			{
+				throw new Exception("You must initialize drawable resources by invoking this in your main Module (replace \"GenericApp\"):\nWindows.UI.Xaml.Media.ImageSource.Drawables = typeof(GenericApp.Resource.Drawable);");
```

Comment:
```suggestion
				throw new Exception("Drawable resources were not initialized.");
```

That error is way out of date 😆

Reply:
True that :-D

Reply:
Can the PR be merged even with the open change request from Jerome? It would be great, I could then finish the JumpList PR 👍 ✔

Reply:
Done :)